scrapy/settings/__init__.py

```diff
@@ -197,6 +198,38 @@ def getdict(self, name, default=None):
             value = json.loads(value)
         return dict(value)
 
+    def getdictorlist(self, name, default=None):
+        """Get a setting value as either an ``OrderedDict`` or a list.
```

Comment:
I wonder if OrderedDict support is needed, given that Scrapy requires Python 3.6+